tools/scripts/rebuildSoundLibraryManifest.rb

```diff
@@ -278,6 +285,16 @@ def build_sound_metadata(sound_objects, previous_metadata)
       begin
         json_response = objects['json'].get
         metadata = JSON.parse(json_response.body.read)
+        aliases = metadata['aliases']
+        # move categories out of aliases
+        categories = []
+        aliases.each do |a|
+          if a.start_with? "category_"
+            categories.push (a.delete_prefix "category_")
+          end
+        end
+        metadata['aliases'] = aliases.map {|a| (a.start_with? "category_") ? (a.delete_prefix "category_") : a}
```

Comment:
this may be an over-optimization, but we can do both of these operations (populating the categories array and mapping the aliases) in the same `.map!`:

```ruby
categories = []
aliases = ["category_background", "alien", "category_fun"]
aliases.map! do |a|
  is_category = a.start_with?("category_")
  a = is_category ? a.delete_prefix("category_") : a
  categories << a if is_category
  a
end

irb(main):037:0> categories
=> ["background", "fun"]
irb(main):038:0> aliases
=> ["background", "alien", "fun"]
```

Reply:
I'm going to leave it as is for now, just because we run this script so rarely. But thanks, that's good to know for the future!